mungegithub/mungers/submit-queue.go

```diff
@@ -499,6 +499,7 @@ func (sq *SubmitQueue) internalInitialize(config *github.Config, features *featu
 		if sq.BatchEnabled {
 			http.Handle("/batch", gziphandler.GzipHandler(http.HandlerFunc(sq.serveBatch)))
 		}
+		// this endpoint is useless without access to prow
```

Comment:
I could figure this out from the if statement below. Why is it useless is what I was more interested in reading...